ios/FluentUI/Shimmer/ShimmerView.swift

```diff
@@ -145,14 +152,12 @@ open class ShimmerView: UIView {
 
 		updateViewCoverLayers()
 
-		guard let containerView = containerView else {
-			return
-		}
+		let viewToCover = containerView ?? self
```

Comment:
containerView is set on init, and we can't do `containerView = self` inside an init, so I do this instead

Reply:
this is bit of behavioral change right? what are the native cases that containerView could be nil?

Reply:
should we just check shimmersLeafViews then viewToCover = self?

Reply:
Theoretically you could have a containerView, AND have shimmersLeafViews = true, so I don't think we would want that.

This is a behavioral change. I don't think there was ever a use case for shimmerView without a containerview. ShimmerLinesView does that, and remains unaffected